src/providerhub/azext_providerhub/generated/action.py

```diff
@@ -20,28 +21,29 @@ def generate_list(inputStr):
     return [word.strip() for word in inputStr.split(',')]
 
 
-class AddCanary(argparse.Action):
+class AddDefaultRolloutROW2(argparse.Action):
     def __call__(self, parser, namespace, values, option_string=None):
         action = self.get_action(values, option_string)
-        namespace.canary = action
+        namespace.rest_of_the_world_group_two = action
 
     def get_action(self, values, option_string):  # pylint: disable=no-self-use
         try:
             properties = defaultdict(list)
             for (k, v) in (x.split('=', 1) for x in values):
                 properties[k].append(v)
             properties = dict(properties)
-        except ValueError as value_error:
-            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string)) from value_error
+        except ValueError:
+            raise CLIError(
+                'usage error: {} [KEY=VALUE ...]'.format(option_string))
         d = {}
         for k in properties:
             kl = k.lower()
             v = properties[k]
-            if kl == 'regions':
-                d['regions'] = v
+            if kl == 'wait-duration':
+                d['wait_duration'] = v[0]
             else:
-                raise CLIError('Unsupported Key {} is provided for parameter canary. All possible keys '
-                               'are: regions'.format(k))
+                raise CLIError('Unsupported Key {} is provided for parameter rest_of_the_world_group_two. All possible keys are: wait-duration'.
+                               format(k))
         return d
```

Comment:
This action only have 'wait-duration' Why not change back to `raw2-wait-duration` to avoid breaking change?

Reply:
Added support for the parameter to avoid breaking change